libs/dh/admin/feature-user-roles/src/lib/create/dh-create-user-role-modal.component.ts

```diff
@@ -205,7 +200,7 @@ export class DhCreateUserRoleModalComponent implements OnInit, AfterViewInit {
       .pipe(takeUntilDestroyed(this.destroyRef))
       .subscribe({
         next: (keys) => {
-          this.eicFunctionOptions = Object.keys(MarketParticipantEicFunction)
+          this.eicFunctionOptions = Object.keys(EicFunction)
```

Comment:
Suggestion: Use our enumToOptions function instead and dhdropdowntranslation directive